cmd/buildkitd/config/config.go

```diff
@@ -40,10 +40,11 @@ type LogConfig struct {
 }
 
 type GRPCConfig struct {
-	Address      []string `toml:"address"`
-	DebugAddress string   `toml:"debugAddress"`
-	UID          *int     `toml:"uid"`
-	GID          *int     `toml:"gid"`
+	Address            []string `toml:"address"`
+	DebugAddress       string   `toml:"debugAddress"`
+	UID                *int     `toml:"uid"`
+	GID                *int     `toml:"gid"`
+	SecurityDescriptor string   `toml:"securityDescriptor"`
```

Comment:
This will be an interesting config option for users to set :smile:, but it's in line with the type of value linux users need to set, and it is correct compared to using group names here.

Reply:
I think we will mostly expect users to use `--group` flag.